otter/test/integration/test_lib.py

```diff
@@ -0,0 +1,137 @@
+"""
+Tests for the lib functions for convergence black-box testing.
+"""
+
+from twisted.internet.defer import Deferred
+from twisted.trial.unittest import SynchronousTestCase
+
+from otter.test.integration import lib
+
+
+class IdentityV2Tests(SynchronousTestCase):
+    """Tests for the :class:`IdentityV2` class."""
+
+    def test_missing_params(self):
```

Comment:
Non-blocking - this seems like it might be more of a test of `characteristic` instead of the class?  Although I can see how you might want to test that the attributes were created correctly (without a default).